### PR TITLE
Refactor namespaces

### PR DESCRIPTION
**Pull request by chrisandreae**

### CODE REVIEW

File: lib/view_model/active_record/update_operation.rb [hunk 1]
```diff
@@ -162,7 +162,7 @@ def run!(deserialize_context:)
 
       @run_state = RunState::Run
       viewmodel
-    rescue ActiveRecord::StatementInvalid, ActiveRecord::InvalidForeignKey, ActiveRecord::RecordNotSaved => ex
+    rescue ::ActiveRecord::StatementInvalid, ::ActiveRecord::InvalidForeignKey, ::ActiveRecord::RecordNotSaved => ex
```
Comment (thefloweringash) [inline lib/view_model/active_record/update_operation.rb]: This conflict is slightly unfortunate.

File: test/unit/view_model/active_record/controller_test.rb [hunk 2]
```diff
@@ -107,8 +107,8 @@ def test_show_missing
     assert_equal(404, parentcontroller.status)
     assert_equal({ 'errors' => [{ 'status' => 404,
                                   'detail' => "Couldn't find Parent with 'id'=9999",
-                                  'type'   => "Deserialization.NotFound",
-                                  "metadata" => { "nodes" => [{ "_type" => "Parent", "id" => 9999 }]}}]},
+                                  'code'   => "Deserialization.NotFound",
+                                  "meta" => { "nodes" => [{ "_type" => "Parent", "id" => 9999 }]}}]},
```
Comment (thefloweringash) [inline test/unit/view_model/active_record/controller_test.rb]: while you're here: any reason to use `"` over `'` for one field?

File: test/unit/view_model/active_record/has_many_through_poly_test.rb [hunk 3]
```diff
@@ -225,8 +225,8 @@ class RenameTest < ActiveSupport::TestCase
     def before_all
       super
 
-      ActiveRecordViewModel::HasManyThroughPolyTest.build_tag_a(self)
-      ActiveRecordViewModel::HasManyThroughPolyTest.build_tag_b(self)
+      ViewModel::ActiveRecord::HasManyThroughPolyTest.build_tag_a(self)
```
Comment (thefloweringash) [inline test/unit/view_model/active_record/has_many_through_poly_test.rb]: While you're here: `self.class`?

File: test/unit/view_model/active_record/controller_test.rb (hunk 2 above)
Comment (chrisandreae) [inline test/unit/view_model/active_record/controller_test.rb]: none at all

File: test/unit/view_model/active_record/has_many_through_poly_test.rb (hunk 3 above)
Comment (chrisandreae) [inline test/unit/view_model/active_record/has_many_through_poly_test.rb]: This is lexically in a nested class, `self.class` is `RenameTest`

**Pull request merged**